x/burn/keeper/burn.go

```diff
@@ -20,23 +24,20 @@ func (k Keeper) BurnCoins(ctx sdk.Context, acc string) error {
 
 	ctx.Logger().Info("find burn coin.", fmt.Sprintf("address: %s, coins: %s", acc, burnCoins))
 
-	err = k.bankKeeper.SubtractCoins(ctx, burnAcc, burnCoins)
+	err = k.bankKeeper.SendCoinsFromAccountToModule(ctx, burnAcc, types.ModuleName, burnCoins)
```

Comment:
Thank you for your comment.
Fortunately(?), it is not allowed to send coins to burn module account, because it is blocked in `x/bank` module.
So it seems work for now, and the only way to burn is to send coins to `burnAcc` (`panacea100000000000000000000000000000000nqmafp`)

Reply:
Goood!